spec/models/comment_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'Comment model' do
+    subject { Comment.new(text: 'test post 1', user_id: 1, post_id: 1) }
+
+    before { subject.save }
+
+    it 'check if it increases the comments' do
```

Comment:
- [ ] It would be better to add negative test cases to our test. This will ensure that the app works as expected

Reply:
Thank you